scripts/check_build_config.js

```diff
@@ -0,0 +1,16 @@
+const buildConfig = require('../src/common/config/buildConfig');
+
+function validateBuildConfig(config) {
+  if (config.enableServerManagement === false && config.defaultTeams && config.defaultTeams.length === 0) {
+    return {
+      result: false,
+      message: `When "enableServerManagement: false" is specified in buildConfig.js, "defaultTeams" must have one team at least.\n${JSON.stringify(config, null, 2)}`
```

Comment:
Propose
```
Specify at least one server for "defaultTeams" in buildConfig.js when "enableServerManagement is set to false.\n${JSON.stringify(config, null, 2)}
```

Would you have an example of what `${JSON.stringify(config, null, 2)}` will output in the console?

Reply:
It would be a formatted text of buildConfig:

```
{
  defaultTeams: [],
  helpLink: 'https://docs.mattermost.com/help/apps/desktop-guide.html',
  enableServerManagement: false
}
```